### PR TITLE
External table SAS and key options

### PR DESCRIPTION
**Pull request by ohadbitt**
External table SAS and key options

### CODE REVIEW

**Comment by PRMerger9** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: data-explorer/external-table.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ The **Create external table** window opens with the **Destination** tab selected
 
 ### Source tab
 
-1. In **Link to source**, enter the [SAS URL](kusto/api/connection-strings/storage.md#generate-a-sas-for-azure-storage-blob-container) of your source container. You can add up to 10 sources. 
+1. In **Link to source**, enter an [account key or SAS URL](kusto/api/connection-strings/storage.md#generate-a-sas-for-azure-storage-blob-container) to your source containerwith read and list permissions. You can add up to 10 sources.
```
Comment (YaelSchuster) [inline data-explorer/external-table.md:50]:
```suggestion
1. In **Link to source**, enter an [account key or SAS URL](kusto/api/connection-strings/storage.md#generate-a-sas-for-azure-storage-blob-container) to your source container with read and list permissions. You can add up to 10 sources. 
```

**Comment by PRMerger15** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by YaelSchuster** [comment]
approved by ADX content team
#sign-off

**Comment by PRMerger12** [comment]
@YaelSchuster: I'm sorry - only the author of this article, @orspod, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Pull request merged**